src/buf.gen.yaml

```diff
@@ -4,7 +4,7 @@ managed:
   go_package_prefix:
     default: github.com/DefangLabs/defang/src/protos
 plugins:
-  - plugin: buf.build/protocolbuffers/go
+  - plugin: buf.build/protocolbuffers/go:v1.36.3
```

Comment:
should we allow patch level upgrades (for CVEs etc?)

Reply:
The only documentation I have been able to find about how to actually pin versions is this line:

> <plugin-version> is optional. If it isn't present, the latest version is used. If it's specified, the revision field can be > specified to pin an exact version.
>
>    The plugin version is specified by the upstream project.
>    The revision is a sequence number that Buf increments when rebuilding or repackaging the plugin.

https://buf.build/docs/configuration/v2/buf-gen-yaml/

This suggests to me that inexact version pinning is not supported.